elections/tools/README.md

```diff
@@ -0,0 +1,37 @@
+
+# Election Tools
+
+This directory contains the tools required to generate the kata electorate
+
+## generate_electorate.py
+
+This tool need the following python libraries pytz github3.py pyyaml.  To
+install them in a virtutal envoronment do something like:
+
+```
+python3 -m venv .venv
+.venv/bin/pip install pytz github3.py pyyaml
+```
+
+Before running the tool you will need to create a
+[github API token](https://github.blog/2013-05-16-personal-api-tokens/)
+
+replace ``__API_TOKEN__`` with your personal token
```

Comment:
> Replace `__API_TOKEN__` with your personal token.

You could use `sed -i` here, but this is fine and I suspect you've specified it like this to maximise security on a multi-user system (by avoiding `ps` seeing the token value).